enhancements/api-review/azure_user_defined_tags.md

```diff
@@ -0,0 +1,327 @@
+---
+title: azure_user_defined_tags
+authors:
+  - "@bhb"
+reviewers:
+  - "@patrickdillon" ## reviewer for installer component
+  - "@JoelSpeed" ## reviewer for api and machine-api-provider-azure components
+  - "@dmage" ## reviewer for cluster-image-registry-operator component
+  - "@Miciah" ## reviewer for cluster-ingress-operator component
+  - "@akhil-rane" ## reviewer for cloud-credential-operator component
+  - "@trozet" ## reviewer for cloud-network-config-controller component
+approvers:
+  - "@jerpeter1" ## approver for CFE
+api-approvers:
+  - "@JoelSpeed" ## approver for api component
+creation-date: 2022-07-12
+last-updated: 2022-07-12
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-8155
+  - https://issues.redhat.com/browse/CORS-2249
+see-also:
+  - "enhancements/api-review/custom-tags-aws.md"
+replaces:
+  - N/A
+superseded-by:
+  - N/A
+---
+
+# Apply user defined tags to all Azure resources created by OpenShift
+
+## Summary
+
+This enhancement describes the proposal to allow an administrator of Openshift to 
+have the ability to apply user defined tags to those resources created by Openshift 
+in Azure.
+
+## Motivation
+
+Motivations include but are not limited to:
+
+- Allow admin, compliance, and security teams to keep track of assets and objects 
+  created by OpenShift in Azure.
+
+### User Stories
+
+- As an openshift administrator, I want to have tags added to all resources created 
+  in Azure by Openshift, so that I can restrict access granted to an OpenShift specific account.
+
+### Goals
+
+- The administrator or service (in the case of Managed OpenShift) installing OpenShift 
+  can configure allowed number of user-defined tags in the Openshift installer generated
```

Comment:
Yes, it's the latter, where an administrator or service can configure tags up to a maximum allowed by the OpenShift..

Reply:
Maybe this would be clearer?
```suggestion
- The administrator or service (in the case of Managed OpenShift) installing OpenShift 
  can configure a list of up to 10 user-defined tags in the OpenShift installer
```
But if I'm the only person who finds the original wording confusing, then don't worry about it.  * grin *.

Reply:
comment incorporated